Gruntfile.js

```diff
@@ -163,7 +164,7 @@ module.exports = function (grunt) {
       },
       ngdocs: {
         options: {
-          title: 'Angular Patternfly Documentation',
+          title: 'ANGULAR',
```

Comment:
Sorry Jeff, if do that, the heading will become "Patternfly - Angular Patternfly", Don't you feel it will be wired?